recipe/migrations/azure_core_cpp_1150.yaml

```diff
@@ -0,0 +1,21 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for azure_core_cpp 1.15.0 and latest storage packages
+  kind: version
+  migration_number: 1
+  exclude_pinned_pkgs: false
```

Comment:
Ah, great, you found this option already! Indeed, if you're adding all the versions for clarity (and expect PRs for places where the version needs to be bumped), then you need this, otherwise the feedstocks pinned below don't get PRs.